### PR TITLE
Enhance: Inline variable in CA1854 (IDictionary.TryGetValue)

### PR DESCRIPTION
**Pull request by Poker-sang**
When simplifying code with `TryGetValue`, we can inline the `out` variable.

Before:
```cs
if (data.TryGetValue(key, out string value))
{
    var a = value;
}
```

After:
```cs
if (data.TryGetValue(key, out string a))
{
}
```

### CODE REVIEW

File: src/NetAnalyzers/CSharp/Microsoft.NetCore.Analyzers/Performance/CSharpPreferDictionaryTryMethodsOverContainsKeyGuardFixer.cs [hunk 1]
```diff
@@ -58,19 +58,26 @@ public override async Task RegisterCodeFixesAsync(CodeFixContext context)
             var dictionaryAccessors = new List<SyntaxNode>();
             ExpressionStatementSyntax? addStatementNode = null;
             SyntaxNode? changedValueNode = null;
+            string? valueName = null;
```
Comment (buyaa-n) [inline src/NetAnalyzers/CSharp/Microsoft.NetCore.Analyzers/Performance/CSharpPreferDictionaryTryMethodsOverContainsKeyGuardFixer.cs:61]: NIT: maybe change into `variableName`?

**Review by buyaa-n — CHANGES_REQUESTED** [review]
Thank you @Poker-sang, the changes looks good, could you please update the [VB fixer](https://github.com/dotnet/roslyn-analyzers/blob/ab13ac75d1195a27e3cf7fb990fc3a51615caa95/src/NetAnalyzers/VisualBasic/Microsoft.NetCore.Analyzers/Performance/BasicPreferDictionaryTryMethodsOverContainsKeyGuardFixer.vb) too? Now this only work on C# but not in VB.

**Comment by Poker-sang** [comment]
@buyaa-n Thank you for the review. I've never written VB, but I tried writing VB fixer anyway, so I'm not sure if I took all the cases into account. Please let me know if there are any problems.

**Comment by buyaa-n** [comment]
> I've never written VB, but I tried writing VB fixer anyway, so I'm not sure if I took all the cases into account. Please let me know if there are any problems.

Thanks, most contributors including myself are same, no previous VB experience, https://converter.telerik.com/ have been helpful for converting code and tests in VB, though looks you don't need it now. Changes looks good

File: src/NetAnalyzers/VisualBasic/Microsoft.NetCore.Analyzers/Performance/BasicPreferDictionaryTryMethodsOverContainsKeyGuardFixer.vb [hunk 2]
```diff
@@ -98,17 +112,20 @@ Namespace Microsoft.NetCore.VisualBasic.Analyzers.Performance
...
 
                         Dim tryGetValueAccess = generator.MemberAccessExpression(containsKeyAccess.Expression,
                                                                                  TryGetValue)
                         Dim keyArgument = containsKeyInvocation.ArgumentList.Arguments.FirstOrDefault()
                         Dim valueAssignment =
                                 generator.LocalDeclarationStatement(dictionaryValueType,
-                                                                    Value,
-                                                                    generator.DefaultExpression(dictionaryValueType)).
+                                                                    variableName,
+                        generator.DefaultExpression(dictionaryValueType)).
```
Comment (buyaa-n) [inline src/NetAnalyzers/VisualBasic/Microsoft.NetCore.Analyzers/Performance/BasicPreferDictionaryTryMethodsOverContainsKeyGuardFixer.vb:125]: NIT: fix alignment
```suggestion
                                                                    variableName,
                                                                    generator.DefaultExpression(dictionaryValueType)).
````

**Review by buyaa-n — APPROVED** [review]
Looks good, thanks!

**Comment by Poker-sang** [comment]
> Thanks, most contributors including myself are same, no previous VB experience, https://converter.telerik.com/ have been helpful for converting code and tests in VB, though looks you don't need it now. Changes looks good

Thanks. I use <https://sharplab.io/> to see the C#/VB syntax tree. The site you recommended looks good too, I'll try it.

**Pull request merged**